Dockerfile

```diff
@@ -13,9 +13,10 @@ RUN mkdir -p ${DIR_PATH}
 WORKDIR ${DIR_PATH}
 ADD . ${DIR_PATH}
 
-ENV CHAIN_URL=http://beta.digitransit.fi/raildigitraffic2gtfsrt/hsl
-ENV TRIP_UPDATE_URL=http://beta.digitransit.fi/hslalert/
-ENV HSL_URL=http://beta.digitransit.fi/navigator-server/siriaccess/vm/json?operatorRef=HSL
+ENV CHAIN_URL=http://api.digitransit.fi/realtime/raildigitraffic2gtfsrt/v1/hsl
```

Comment:
Could we rename `CHAIN_URL` to something a bit more meaningful while doing this?